docs/application/dotnet/guides/app-management/app-manager.md

```diff
@@ -113,7 +135,37 @@ To get information on filtered applications:
     }
     ```
 
+<a name="manage_current"></a>
+## Getting Information on Current Application
+
+To get information on the current application, follow these steps:
+
+1.  Call the `Current` property of the [Tizen.Applications](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.ApplicationInfoFilter.html) class:
+
+    ```
+    Application application = Application.Current;
+    ```
+
+2.  Operate on the application context:
```

Comment:
Is this "context" correct in this sentence? In this section it looks like we are talking about Application information itself, not context. Would you please check?

Reply:
I completely agree with you. To be more specific, can we rephrase as:
Operate on the application information:

Please verify.

Reply:
Done